tests/testsuite/new.rs

```diff
@@ -509,3 +509,25 @@ fn git_default_branch() {
     let head = repo.find_reference("HEAD").unwrap();
     assert_eq!(head.symbolic_target().unwrap(), "refs/heads/hello");
 }
+
+#[cargo_test]
+fn non_utf8_str_in_ignore_file() {
+    let gitignore = paths::home().join(".gitignore");
+    File::create(gitignore).unwrap();
+
+    fs::write(paths::home().join(".gitignore"), &[0xFF, 0xFE]).unwrap();
+    print!("{}", paths::home().display());
```

Comment:
Could you please explain why we need this print?

Reply:
Sorry, this is just for local testing. Removed it. Thank you.